### PR TITLE
Implement EZP-23993: Clarify and document how security issues should be reported

### PR DESCRIPTION
**Pull request by glye**
https://jira.ez.no/browse/EZP-23993

Ref ezsystems/ezpublish-kernel#1201

### CODE REVIEW

**Comment by pspanja** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -23,3 +23,9 @@ index. After the index is complete, the copy of the current index is replaced
 with the newest version. In addition, the search engine remembers all caching
 structures from previous searches and, during indexing, updates these as well.
 Therefore, the more the search engine is used, the faster it becomes.
+
+
+Issue tracker
+-------------
+Submitting bugs, improvements and stories is possible on https://jira.ez.no/browse/EZP
```
Comment (bdunogier) [inline README.md]: Dots at the end of sentences wouldn't hurt.

**Comment by yannickroger** [comment]
+1

File: README.md (hunk 1 above)
Comment (glye) [inline README.md]: @bdunogier Except when they break URLs. Are you saying the formatting is smart enough to handle an appended dot?

**Comment by andrerom** [comment]
+1

**Comment by glye** [comment]
@bdunogier Dots added, nothing exploded :)

**Comment by bdunogier** [comment]
> @bdunogier Dots added, nothing exploded :)

Capital :-)

**Pull request merged**